validation/src/index.js

```diff
@@ -50,19 +55,25 @@ const validate = async () => {
 	).map((fileName) => path.join(config.rootDir, fileName));
 
 	// Validate if app.json and nativetoken.json is present anywhere except networkDir/appName/
-	await validateConfigFilePaths(changedAppFiles);
+	await validateConfigFilePaths(changedAppFiles, validationErrors);
 
 	// Validate all app.json and nativetoken.json schemas
-	await validateAllSchemas(changedAppFiles);
+	await validateAllSchemas(changedAppFiles, validationErrors);
 
 	// Check whitelisted files in all changed network directories
-	await validateAllWhitelistedFiles(allChangedFiles);
+	await validateAllWhitelistedFiles(allChangedFiles, validationErrors);
 
 	// Check for config files in all changed apps in networks directories
-	await validateAllConfigFiles(changedAppDirs);
+	await validateAllConfigFiles(changedAppDirs, validationErrors);
 
 	// Validate serviceURLs
-	await validateURLs(changedAppFiles);
+	await validateURLs(changedAppFiles, validationErrors);
+
+	if (validationErrors.length > 0 && !doesAutherBelongToOrg) {
+		throw new Error(validationErrors.join('\n'));
+	} else if (validationErrors.length > 0) {
+		console.warn(validationErrors.join('\n'));
+	}
```

Comment:
We should throw errors whenever there's a validation error. But with `isAuthorFromDevTeam: true`, we can skip the `validateAllWhitelistedFiles` invocation on L64.

Also, please verify if `isAuthorFromDevTeam` is a boolean value of a string. Most likely, it is a string.

```suggestion
	if (validationErrors.length > 0) {
		throw new Error(validationErrors.join('\n'));
	}
```